Update file to add note on site column name change (#3611)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/add-columns-content-type.md

7. Select **Save**.

> [!NOTE]

> >  In case of updating column name for an existing site column that is part of a published content type from the content type hub, please ensure that the language of the target site and language of the browser are same for the latest site column name to appear after correctly after the sync completes.

Reviewer: ```suggestion
>  When you update the column name of an existing site column that is part of a published content type from the content type hub, ensure that the language of the target site and the language of the browser are the same for the latest site column name to appear correctly after the sync completes.
```
